tests/testthat/test-z-integration.R

```diff
@@ -448,3 +448,18 @@ test_that("Can create a report with parameters", {
     readRDS(file.path(tmp, id, "meta", "info.rds"))$parameters,
     list(time = 10, poll = 1))
 })
+
+test_that("Can get dependencies", {
+  server <- start_test_server()
```

Comment:
Should we test with a report with more dependencies than `count` report?

You should be able to start test server with more reports by using
```
path <- orderly_git_example("demo")
server <- start_test_server(path)
```
which has reports, `use_dependency` or `use_dependency_2`

Reply:
Actually just seen `test-api-endpoints.R` covers this case.